minor edits to the Introduction (#117)
ietf-wg-ccwg/rfc5033bis: draft-ietf-ccwg-rfc5033bis.md

> of a proposed congestion control algorithm at the IETF. It replaces RFC5033 to reflect
> changes in the congestion control landscape.
> 
> This document obsoletes RFC 5033.

Reviewer: ```suggestion
of a proposed congestion control algorithm at the IETF. It obsoletes RFC5033 to reflect
changes in the congestion control landscape.
```

a proposal is appropriate for publication in the RFC series and for
deployment in the Internet.

> This document obsoletes {{?RFC5033}} which was

Reviewer: ```suggestion
This document obsoletes {{?RFC5033}}, which was
```

published in 2007 as a Best Current Practice to evaluate proposed
congestion control algorithms as Experimental or Proposed Standard RFCs.

The IETF's standard congestion control algorithms have been shown to have
performance challenges in various environments

> (e.g., high-speed networks, cellular and Wireless Local Area Network (WLAN) technologies, and

Reviewer: ```suggestion
(e.g., high-speed networks, cellular and WiFi wireless technologies, and
```

long distance satellite links)
and also flows carrying specific workloads (Voice over IP (VoIP), gaming, and videoconferencing).

> In 2007, TCP {{?RFC9293}} was the dominant consumer of this work on congestion control, and proposals

Reviewer: ```suggestion
In 2007, TCP {{?RFC9293}} was the dominant consumer of this work, and proposals
```

were typically discussed in the Internet Congestion Control Research Group (ICCRG).
The Datagram Congestion Control Protocol (DCCP) {{?RFC4340}}
was developed as a method for defining new congestion control algorithms for
datagram traffic.

Since {{?RFC5033}} was published, many conditions have changed.
The set of protocols using these algorithms has spread beyond
TCP, Stream Control Transmission Protocol (SCTP) {{?RFC9260}}, and DCCP, to include QUIC {{?RFC9000}}, RTP Media Congestion Avoidance Techniques (RMCAT) (e.g., {{?RFC8836}}) and beyond.

Some proponents of alternative congestion control algorithms now have the opportunity
to test and deploy at scale without IETF review.
There is more interest in specialized use cases, such as data centers (e.g., {{?RFC8257}}), and in
support for a variety of upper layer protocols/applications, e.g.,
real-time protocols.
Finally, the community has gained much more experience with indications

> of congestion beyond indicators based on packet loss.

Reviewer: ```suggestion
of congestion beyond packet loss.
```

deployment of new algorithms is not in fact gated by publication
of the algorithm as an RFC.

> Nevertheless, editing and publicly sharing the specification of congestion control algorithms has a number of advantages:

Reviewer: ```suggestion
Nevertheless, specifying congestion control algorithms has a number of advantages:
```

- A specification can help implementers, operators, and other interested
  parties to develop a shared understanding of how the algorithm works and how
  it is expected to behave in various different scenarios or configurations.

> - A specification can help potential contributors understand the algorithm and objectives,

Reviewer: ```suggestion
- A specification can help potential contributors understand the algorithm,
```

the proposed congestion control algorithm. The community should also consider
other non-standard congestion control algorithms that are known to be widely deployed.

> This guideline is not a requirement for strict Reno- or Cubic-

Reviewer: ```suggestion
Note that this guideline is not a requirement for strict Reno- or Cubic-
```
